sensor_msgs/Image[] images
Rect[] rects
string[] labels
---
bool ok
string error
